src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/OrderMadeDefendantNotificationHandler.java

```diff
@@ -39,7 +32,8 @@ public class OrderMadeDefendantNotificationHandler extends OrderCallbackHandler
 
     private static final List<CaseEvent> EVENTS = List.of(CREATE_DASHBOARD_NOTIFICATION_FINAL_ORDER_DEFENDANT,
                                                           CREATE_DASHBOARD_NOTIFICATION_DJ_SDO_DEFENDANT,
-                                                          CREATE_DASHBOARD_NOTIFICATION_SDO_DEFENDANT);
+                                                          CREATE_DASHBOARD_NOTIFICATION_SDO_DEFENDANT,
+                                                          UPDATE_DASHBOARD_TASK_LIST_UPLOAD_DOCUMENTS_FINAL_ORDERS);
```

Comment:
Why was UPDATE_DASHBOARD_TASK_LIST_UPLOAD_DOCUMENTS_FINAL_ORDERS added?
Could you not just use CREATE_DASHBOARD_NOTIFICATION_FINAL_ORDER_DEFENDANT?

Did you make any similar changes for claimant?

Reply:
I have choose different approach at the end, please have a look.